tasks/usemin.js

```diff
@@ -105,19 +105,35 @@ module.exports = function (grunt) {
 
   grunt.registerMultiTask('usemin', 'Replaces references to non-minified scripts / stylesheets', function () {
     var debug = require('debug')('usemin:usemin');
+    var type = this.target;
+    // If type is not html/css/js and the customized configuration provides 'type' in options. Below is the example.
```

Comment:
@markfredchen should go to README.

`If type is not html/css/js` -> `If type is different from html/css/js`